contents/handbook/people/compensation.md

```diff
@@ -45,6 +45,14 @@ Within each level, we believe there's a place to have incremental steps. We defi
 ​
 You'll receive approximately 0.1-0.5% of the company with a standard 1-year cliff, 4-year vest. Broadly, this will depend on the Level as per the Experience Factor. We may change this policy from time to time depending on our rate of hiring - e.g. if we had a gap in hiring for an extended period, we would adjust this.
 ​
+Whilst the terms of options for *any company* could vary if we were ever acquired, we have set them up with the following key terms:
+
+* 10 year exercise period
```

Comment:
I'd say '10 years to exercise your options in the event that you leave Posthog.'

Reply:
fixed